components/eam/src/physics/crm/samxx/crm_variance_transport.cpp

```diff
@@ -113,6 +111,8 @@ void VT_diagnose() {
   real2d t_mean("t_mean", nzm, ncrms);
   real2d q_mean("q_mean", nzm, ncrms);
 
+  int idx_qt = index_water_vapor;
```

Comment:
Is this correct? `idx_qt` should be index to total water, but it's being set to something called `index_water_vapor`.

Reply:
Yea, this whole thing is super confusing. index_water_vapor is used to point to the index of the prognostic variable that contains the vapor, even if it's total water. I did it this way to be consistent with other parts of the code, which are already inconsistent and confusing. What's more confusing is that for P3 we have a "idx_qt" to match the form of the other tracer indices like "idx_qc". However, even though total water contains the cloud liquid water, we still have to advect a separate "diagnostic" cloud liquid water tracer. So now I'm thinking that the code would be much more sensible if we just advected vapor and cloud water separately, but we'd still need to use total water for the 1-mom option. 

In this case I decided to mimic what is done in the mean-state acceleration routine. It's "correct" here because index_water_vapor=0, which is the micro_field index of the main prognostic total water tracer.